include/psa/crypto_values.h

```diff
@@ -1501,6 +1504,18 @@
  */
 #define PSA_KEY_LIFETIME_PERSISTENT             ((psa_key_lifetime_t)0x00000001)
 
+#define PSA_KEY_LIFETIME_IS_VOLATILE(lifetime) \
```

Comment:
Please refer to the definition of `PSA_KEY_LIFETIME_IS_VOLATILE` in https://github.com/ARMmbed/mbed-crypto/pull/358, which is slated to be included in the next release of Mbed Crypto and in an upcoming version of the PSA Cryptography specification.

Reply:
I am fine with that new definition. Im not quite sure how to get it in here though.